### ISSUE TITLE
MD027 - Indented code blocks trigger warning

### ISSUE THREAD

**Issue by garazdawi** [DavidAnson/markdownlint#1393]
When writing the following:

```
> This is some code:
>
>     Some indented code
>
>     More code
```

The MD0027 warning is triggered, which I think is a bug as the above is valid markdown.

> This is some code:
>
>     Some indented code
>
>     More code

**Comment by garazdawi** [comment]
Hmm, and just as I hit send I cannot reproduce it any more.... sorry for the noise. I'll reopen if I fugure out what is going on.

**Comment by garazdawi** [comment]
Seems like it is https://github.com/DavidAnson/markdownlint/commit/730ae9a96fb205a99a391fe4a2fc7b9df0a53ad2#diff-2725cc5ae2a7c11d0b5b12c1333d68fe7d2f8d121a1d2f66fb6f1b653146b9b3 that changed the behaviour of this. Not sure if it is already fixed or not. The reason I got confused was that I was running an earlier version locally and a later version in CI.

**Comment by DavidAnson** [comment]
MD027 does produce a warning for the indented code block in your example: https://dlaa.me/markdownlint/#%25m%23%20Issue%201393%0A%0A%3E%20This%20is%20some%20code%3A%0A%3E%0A%3E%20%20%20%20%20Some%20indented%20code%0A%3E%0A%3E%20%20%20%20%20More%20code%0A

While correct, I think it should probably accommodate indented code blocks inside blockquotes. I'll have a look.
